gitlab_test.go

```diff
@@ -89,11 +73,6 @@ func testJsonBody(t *testing.T, r *http.Request, want values) {
 	}
 }
 
-func responseBody(w http.ResponseWriter, filename string) {
-	body, _ := ioutil.ReadFile(filename)
-	w.Write([]byte(body))
-}
-
 func TestNewClient(t *testing.T) {
```

Comment:
Please do not delete these unused test functions. We need to start adding some tests and these helper functions will be needed when we do... So I would like to keep these in.